tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -71,6 +71,7 @@ if [ "$OS_VERSION" = "20.04" ]; then
 	    add-apt-repository -y ppa:deadsnakes/ppa
         apt-get update
         apt-get install -y --no-install-recommends \
+                python${PYTHON_VER}-distutils \
```

Comment:
Why does it need to go to this path? Why not just use the default python version? For example, On Ubuntu 20.04 it should be python 3.8.

Reply:
changed to python 3.8 and removed this package, since it's only needed for python 3.7